### ISSUE TITLE
Update resource e2e tests

### ISSUE THREAD

**Issue by joe-allen-89** [adaptlearning/adapt-contrib-resources#137]
### Subject of the issue/enhancement/features
A core change to drawer (https://github.com/adaptlearning/adapt-contrib-core/pull/663) has swapped aria-expanded for aria-haspopup, the e2e tests currently fail as they're looking for aria-expanded. e2e tests need to be updated to remove aria-expanded. 

### Your environment
* github framework release
